src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/helper/builders/OffenderBuilder.kt

```diff
@@ -12,6 +12,7 @@ class OffenderBuilder(
   var birthDate: LocalDate = LocalDate.of(1965, 7, 19),
   var genderCode: String = "M",
   var bookingBuilders: List<OffenderBookingBuilder> = mutableListOf(),
+  val repository: Repository? = null,
```

Comment:
Again the repository is just passed in for the child course activity - this anti-pattern is acknowledged and we're hoping to move away from it.